implementation/e2e_protection/src/e2e/profile/e2e_provider_impl.cpp

```diff
@@ -178,6 +187,31 @@ e2e_provider_impl::make_e2e_profile_config(const std::shared_ptr<cfg::e2e> &_con
             min_data_length, max_data_length, max_delta_counter);
 }
 
+template<>
+vsomeip_v3::e2e::profile05::profile_config
+e2e_provider_impl::make_e2e_profile_config(const std::shared_ptr<cfg::e2e> &_config) {
+
+    uint32_t data_id = read_value_from_config<uint32_t>(_config, "data_id");
+
+    size_t offset = read_value_from_config<size_t>(_config, "crc_offset");
+    if (offset % 8)
+        VSOMEIP_ERROR << "Offset in E2E P05 configuration must be multiple of 8"
+            " (" << offset << ")";
+    offset /= 8;
+
+    size_t min_data_length = read_value_from_config<size_t>(_config,
```

Comment:
min_data_length and max_data_length are not parameters of profile 05, instead use just dataLength